linq4j/src/main/java/org/apache/calcite/linq4j/EnumerableDefaults.java

```diff
@@ -3845,67 +3850,94 @@ public void close() {
           private final Enumerator<TSource> seedEnumerator = seed.enumerator();
           private Enumerator<TSource> iterativeEnumerator = null;
 
+          // Set to control duplicates, only used if all is false
```

Comment:
Nit: shall we give a double quotes for `all` ?

Reply:
Agree, change committed.
Thanks for the review @jinxing64 !